### DISCUSSION TITLE
how to genrate a single css file

### DISCUSSION THREAD

**Discussion by WingDust** [unocss/unocss#1827] [Q&A]
unocss,how to genrate a single css file

**Comment by chu121su12** [comment]
Calling `uno.generate()` returns object that you can extract its css

https://github.com/unocss/unocss/blob/c93b15d446d2c0c26be3fda9a8af9129ff44fd45/packages/core/src/generator/index.ts#L119-L122

https://github.com/unocss/unocss/blob/c93b15d446d2c0c26be3fda9a8af9129ff44fd45/packages/core/src/types.ts#L637-L638

You should then be able to output the result to a file.
